### ISSUE TITLE
Reader: Add more context to action buttons on posts

### ISSUE THREAD

**Issue by davidakennedy** [wordpress-mobile/WordPress-Android#6071]
### Expected behavior

When I navigate the Reader in the Android app with a screen reader, I expect to find more context on buttons.

### Actual behavior

* I found the "Visit" icon button was unlabeled. The word "Visit" lacked context for a screen reader user. I want to know what I'm visiting. This can be combined into one button, labeled with "Visit Post Title Name" "Post Title Name" can be hidden visually.
* The Likes button should be labeled with Like "Post Title Name" Has 77 Likes. Or Like "Post Title Name" No Likes yet. You could also communicate state here. So if a post is in a liked state, it could say: Like "Post Title Name" Pressed Has 77 Likes (or  No Likes yet).
*  Similarly with the Comments button, it needs a more specific label. Comment on "Post Title Name"
* The More button is labeled, but could use more context. "Follow Site Menu" maybe?

### Steps to reproduce the behavior

* Start Talk Back
* Open WordPress app
* Open Reader.
* Begin navigating a list of post by swiping right to navigate forward, reaching the action button area.

##### Tested on [device], Android [version]

* Tested on Nexus 6P, Version Alpha-60.

**Comment by kwonye** [comment]
Thanks for this, @davidakennedy! We'll get working on this today.
